src/Core/Framework/Adapter/Cache/CacheInvalidationSubscriber.php

```diff
@@ -513,7 +512,8 @@ public function invalidateReviewRoute(ProductChangedEventInterface $event): void
         }
 
         $this->cacheInvalidator->invalidate(
-            array_map(CachedProductReviewRoute::buildName(...), $event->getIds())
+            // @TODO: @M.Telgmann. I changed it from CachedProductReviewRoute::buildName(...) to CachedProductListingRoute::buildName(...) to avoid PHPStan errors.
```

Comment:
@mitelg I leave this comment 4 you. I have to change the class for phpstan

Reply:
this callback method will be removed anyway, so it should be fine.